content/en/docs/reference/access-authn-authz/webhook.md

```diff
@@ -108,7 +108,12 @@ the request and respond to either allow or disallow access. The response body's
 }
 ```
 
-To disallow access, the remote service would return:
+For disallowing access there are two options. The first one is the "no opinion"
+option and should be preferred for most of cases. By setting the 
```

Comment:
I like this approach. Suggested copy:

===

For disallowing access there are two methods.

The first method is preferred in most cases, and indicates the authorization webhook does not allow the request, but if other authorizers are configured, they are given a chance to allow the request. If there are no other authorizers, or none of them allow the request, the request is forbidden. The webhook would return:

```json
{
  "apiVersion": "authorization.k8s.io/v1beta1",
  "kind": "SubjectAccessReview",
  "status": {
    "allowed": false,
    "reason": "user does not have read access to the namespace"
  }
}
```

The second method denies immediately, short-circuiting evaluation by other configured authorizers. This should only be used by webhooks that have detailed knowledge of the full authorizer configuration of the cluster. The webhook would return:

```json
{
  "apiVersion": "authorization.k8s.io/v1beta1",
  "kind": "SubjectAccessReview",
  "status": {
    "allowed": false,
    "denied": true,
    "reason": "user does not have read access to the namespace"
  }
}
```

Reply:
Looks great to me! Only thing I think might be good to add is to state explicitly that the first option is sometimes referred to as the "no opinion" option - as this is what it's called elsewhere in the documentation, having it at least referenced here would connect the dots somewhat. What do you think?

Reply:
Sure, sounds good.

Reply:
Great! How do we proceed? Haven't worked with a documentation PR like this before. Should I update this one with the copy above + the no opinion reference or how is it done?

Reply:
yes, just update this PR with the desired text, and squash down to a single commit for review

Reply:
Done. Thanks for guiding me through this @liggitt 
Much appreciated 👍